src/components/about/About.component.tsx

```diff
@@ -0,0 +1,40 @@
+import aboutImg from '@/assets/aboutImg.png';
+
+import './about.css';
+
+export const AboutComponent = () => (
+    <section className="about">
+        <div className="about__descr">
+            <h1 className="about__descr-title">About Me</h1>
+            <p className="about__descr-text">
+                Hi! My name is Artem Sokur and I&apos;m a Junior Frontend Developer. I am already familiar with main Web Technologies like
+                React, HTML, CSS, JavaScript and Git version control system.
+            </p>
+            <p className="about__descr-text">
+                This page was developed during the course{' '}
+                <a className="intro" href="https://www.mastersacademy.education/frontend-react-it" target="_blank" rel="noreferrer">
```

Comment:
Considering the repeated use of `<a>` tags with identical functionality and styling throughout the application, it would be beneficial to encapsulate this pattern into a reusable component. 
This approach will centralize common logic and styles, facilitating easier updates and ensuring consistency across the application. Please create a helper component for the `<a>` tags to streamline code maintenance and improve readability.

```tsx
<Link href target ...etc></Link>
```
component should be able accept any props which native `a` tag can accept

Hint: Go to definiton files of a tag you can do it by `ctrl + click` or `cmd + click` on MacOs
https://mortenbarklund.com/blog/react-typescript-props-spread/
![Screenshot 2024-04-19 at 18 10 36](https://github.com/Allocene/fe-react-2024/assets/55930010/e9051de3-4049-4ac1-b776-ab02dbe7ffc2)